spec/views/people/index_spec.rb

```diff
@@ -0,0 +1,7 @@
+
+xdescribe 'Check if people are listed', type: :system do
```

Comment:
is this test related to the current feature?

Reply:
No